packs/data/feats.db/fledgling-flight.json

```diff
@@ -21,11 +21,17 @@
         "level": {
             "value": 5
         },
-        "location": "",
+        "location": null,
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "speed",
+                "key": "RollOption",
+                "option": "fledgling-flight"
+            }
+        ],
```

Comment:
Not needed. Feats all add their own roll options already.

Reply:
I could have sworn I tested this and it didn't have a roll option! I'll remove it